frontend/pages/policies/ManagePoliciesPage/components/PoliciesListWrapper/PoliciesListWrapper.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { noop } from "lodash";
+
+import Button from "components/buttons/Button";
+import { IPolicy } from "interfaces/policy";
+import TableContainer from "components/TableContainer";
+import { generateTableHeaders, generateDataSet } from "./PoliciesTableConfig";
+
+const baseClass = "policies-list-wrapper";
+const noPoliciesClass = "no-policies";
+
+interface IPoliciesListWrapperProps {
+  policiesList: IPolicy[];
+  isLoading: boolean;
+  onRemovePoliciesClick: any;
```

Comment:
Let's type this. Start with `() => void` and fill in the parameters or return type if your function calls for them.

Reply:
I'm having a similar typing issue here as in `AddPolicyModal`